code/TestAutomation.Core/Browser/WebDriverExtensions.cs

```diff
@@ -0,0 +1,222 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using TestAutomation.Core.Elements;
+using TestAutomation.Utilities;
+
+namespace TestAutomation.Core.Browser
+{
+    public static class WebDriverExtensions
+    {
+        public static TimeSpan DefaultTimeout = TimeSpan.FromMilliseconds(15000);
+        public static TimeSpan DefaultSleepTimeout = TimeSpan.FromMilliseconds(1000);
+
+        public static string GetUrl(this IWebDriver webDriver)
+        {
+            return webDriver.Url;
+        }
+
+        public static IWebDriver GotToWebPageUrl(this IWebDriver webDriver, string url)
+        {
+            webDriver.Navigate().GoToUrl(url);
+            return webDriver;
+        }
+
+        public static void ScrollToElement(this IWebDriver webDriver, IWebElement element)
+        {            
+            Actions actions = webDriver.GetActions();
+            actions.ScrollToElement(element).Build().Perform();
+        }
+        
+        public static void GoBack(this IWebDriver webDriver)
+        {
+            Logger.Info("Navigate Back");
+            webDriver!.Navigate().Back();
+        }
+
+        public static void GoForward(this IWebDriver webDriver)
+        {
+            Logger.Info("Navigate Back");
+            webDriver!.Navigate().Forward();
+        }
+
+        public static void RefreshPage(this IWebDriver webDriver)
+        {
+            Logger.Info("Refresh page");
+            webDriver.Navigate().Refresh();
+        }
+        public static void Maximize(this IWebDriver webDriver)
+        {
+            Logger.Info("Maximize Browser");
+            webDriver.Manage().Window.Maximize();
+        }
+        public static void ScrollTop(this IWebDriver webDriver)
+        {
+            Actions actions = webDriver.GetActions();
+            actions.SendKeys(Keys.Home).Build().Perform();
+        }        
+       
+        public static void SetSessionToken(this IWebDriver webDriver, string token)
+        {
+            var tokenValue = "{\"type\":\"bearer\",\"value\":\"" + token + " \"}";
+            ExecuteScript(webDriver, "javascript:localStorage.token=arguments[0];", tokenValue);
+        }
+
+        public static object ExecuteScript(this IWebDriver webDriver, string script, params object[] args)
```

Comment:
It is better to split methods to different classes. 1 general helper, 1 JS helper and 1 Actions helper :)

Reply:
I will take a look on it